tests/keras_tests/feature_networks_tests/test_features_runner.py

```diff
@@ -788,9 +790,14 @@ def test_keras_tpcs(self):
         TpcTest(f'{C.IMX500_TP_MODEL}.v2_lut', self).run_test()
         TpcTest(f'{C.IMX500_TP_MODEL}.v3', self).run_test()
         TpcTest(f'{C.IMX500_TP_MODEL}.v3_lut', self).run_test()
+        TpcTest(f'{C.IMX500_TP_MODEL}.v4', self).run_test()
         TpcTest(f'{C.TFLITE_TP_MODEL}.v1', self).run_test()
         TpcTest(f'{C.QNNPACK_TP_MODEL}.v1', self).run_test()
 
+    def test_16bit_activations(self):
```

Comment:
Also I would add function tests checking the set config function separately for filtered candidates (not a part of a complete MST execution)